src/detect-bsize.c

```diff
@@ -291,6 +293,38 @@ static int DetectBsizeSetup (DetectEngineCtx *de_ctx, Signature *s, const char *
     DetectBsizeData *bsz = DetectBsizeParse(sizestr);
     if (bsz == NULL)
         goto error;
+
+    SigMatch *pm = NULL;
+    pm = DetectGetLastSMFromLists(s, DETECT_CONTENT, -1);
```

Comment:
nit: `SigMatch *pm = DetectGetLastSMFromLists(s, DETECT_CONTENT, -1);` Not need to first init separately, followed by unconditionally overwriting the value.